packages/buefy-next/src/components/upload/Upload.vue

```diff
@@ -37,12 +37,20 @@
     </label>
 </template>
 
-<script>
-import CompatFallthroughMixin from '../../utils/CompatFallthroughMixin'
+<script lang="ts">
+import { defineComponent } from 'vue'
+import type { ExtractComponentProps } from '../../utils/helpers'
 import FormElementMixin from '../../utils/FormElementMixin'
 import { File } from '../../utils/ssr'
 
-export default {
+interface BUploadData {
+    newValue: [Object, Function, File, Array<T>, null],
```

Comment:
There are a couple of problems at this line:
1. `T` is not defined
2. `newValue`'s type does not match that of the `modelValue` prop